README.md

```diff
@@ -65,7 +65,18 @@ $ brew install git make jq qemu
 ##### On Ubuntu Linux
 
 ```sh
-$ sudo apt-get install -y git make jq qemu-utils qemu-system-x86 qemu-system-aarch64
+$ sudo apt-get install -y git make jq qemu binfmt-support qemu-user-static \
+    qemu-utils qemu-system-x86 qemu-system-aarch64
+```
+
+#### Setup Docker
+
+##### Enable execution of different multi-architecture containers
+
+This step is required to create eve bootable images with a different architecture than the host architecture. 
```

Comment:
First of all -- please take care of yetus complaints below @Insei 

Also, you need to spell out more clearly that this is only required on Linux, not on Mac OS X.

Reply:
Ok